js/main.js

```diff
@@ -146,3 +170,27 @@ const $formLaunch = document.querySelector('.form-launch');
 $formLaunch.addEventListener('click', function (event) {
   viewSwap('entry-form');
 });
+
+const $ul = document.querySelector('ul');
+
+$ul.addEventListener('click', function (event) {
+  const $li = event.target.closest('li');
+  if (event.target.tagName === 'I') {
+    viewSwap('entry-form');
+    const numEntry = $li.getAttribute('data-entry-id');
+    let clickedEntry = 0;
+    for (let i = 0; i < entriesArray.length; i++) {
+      if (entriesArray[i].entryId === Number(numEntry)) {
+        clickedEntry = entriesArray[i];
+      }
+    }
+    data.editing = clickedEntry;
+
+    $codeJournal.querySelector('#title').value = clickedEntry.userTitle;
+    $codeJournal.querySelector('#photoURL').value = clickedEntry.userPhoto;
+    $codeJournal.querySelector('#notes').value = clickedEntry.userNotes;
+    $url.src = clickedEntry.userPhoto;
+
+    document.querySelector('#title-entry').textContent = 'Edit Entry';
```

Comment:
Move the `querySelector`s into the global space. This way we only need to do the search once, and not every time this function gets called.